.github/workflows/CheckStyleAndLinting.yml

```diff
@@ -0,0 +1,18 @@
+name: Style and Lint Check
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  build_aosm:
+    runs-on: ubuntu-latest
+    container: mcr.microsoft.com/azure-cli/tools:latest
+    permissions: write-all
+    steps:
```

Comment:
The same question as above. Our CI will check style and linter for your extension automatically.

Reply:
I have removed this file now (left it on our pre-release branch only)